src/plain-animator.ts

```diff
@@ -28,27 +29,30 @@ export class PlainAnimator {
    * @param {number} tilesAmountVertically - number of rows in your sprite image
    * @param {number} tilesTotalAmount - number of frames in your sprite image
    * @param {number} framesPerSecond - number of frames per second, for example 15
+   * @param {number} frameOffset - number of frames to skip before the setting the initial frame, default 0
    */
   constructor(
     public texture: Texture,
     protected tilesAmountHorizontally: number,
     protected tilesAmountVertically: number,
     protected tilesTotalAmount: number,
     framesPerSecond: number,
+    protected frameOffset: number,
```

Comment:
it's a super minor issue, but please could you move this so the protected variables will be together? 👍